### ISSUE TITLE
`nvim .` bugged out?

### ISSUE THREAD

**Issue by stankovictab** [nvim-tree/nvim-tree.lua#1661]
### Description

A couple of days ago I updated nvim-tree and suddenly opening nvim into a directory, like with `nvim .` or `nvim ~`, doesn't open nvim-tree automatically anymore, it just opens a strange file with the same name as the directory, and in order to open nvim-tree, I need to do `:NvimTreeToggle`.\
Any ideas on why this is happening?\
I have `vim.g.loaded_netrwPlugin = 1` and `vim.g.loaded_netrw = 1` set, and I have `disable_netrw = true` in the config.

This is the result of running `nvim ~/.dotfiles/`.
![image](https://user-images.githubusercontent.com/62820268/196014727-eeb2d4d3-d508-40ac-a8d8-a12901bc93d5.png)

### Neovim version

```text
NVIM v0.8.0-1210-gd367ed9b2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by runner@fv-az505-656
```

### Operating system and version

Kubuntu 21.10 x86_64, Kernel 5.13.0-52-generic

### nvim-tree version

Latest, updated through Packer

### Minimal config

```lua
-- Don't quite understand how to do this, so here's my nvim-tree config

require('nvim-tree').setup { -- BEGIN_DEFAULT_OPTS
	disable_netrw = true, -- Disables the builtin NeoVim explorer, default false
	-- hijack_netrw = false, -- Default is true
	-- hijack_unnamed_buffer_when_opening = true, -- Default is false
	-- hijack_cursor = true, -- Resets the cursor to the first letter of file names
	renderer = {
		icons = {
			glyphs = {
				default = "",
				symlink = "",
				folder = {
					arrow_closed = "",
					arrow_open = "",
					default = "", -- default = "",
					open = "", -- open = "",
					empty = "",
					empty_open = "",
					symlink = "",
					symlink_open = "",
				},
				git = {
					unstaged = "", -- unstaged = "✗",
					staged = "", -- staged = "✓",
					unmerged = "",
					renamed = "➜",
					untracked = "", -- untracked = "★",
					deleted = "",
					ignored = "◌",
				},
			},
			git_placement = "after", -- Git info goes after name of file or folder
		},
	},
	actions = {
		open_file = {
			quit_on_open = true, -- quit_on_open = false,
		},
	},
}
```

### Steps to reproduce

1. `nvim .`

### Expected behavior

nvim-tree pops up immediately, like before

### Actual behavior

I get a weird folder-file thing that I can write in but can't save, as it's a directory.

**Comment by markmansur** [comment]
Yeah I got the same issue

**Comment by markmansur** [comment]
Looks like passing `open_on_setup=true` fixed it for me

**Comment by stankovictab** [comment]
Oh yeah, that did it! 
The defaults were probably changed in some commit
Thanks!

**Comment by alex-courtis** [comment]
> Looks like passing `open_on_setup=true` fixed it for me

The directory was being opened as an inindented side-effect without `open_on_setup=true`, which was "fixed" via #1618
